addons/sourcemod/scripting/surftimer/sql.sp

```diff
@@ -3427,28 +3427,33 @@ public void db_UpdateReplaysTick(int client, int style){
 	else
 		cp_count = g_TotalStages - 1;
 
+	Transaction TicksTransaction = new Transaction();
+
 	if(!g_bReplayTickFound[style]){
 		g_bReplayTickFound[style] = true;
 		for(int i = 0; i < cp_count; i++){
 			Format(szQuery, sizeof(szQuery), sql_insertReplayCPTicks, g_szMapName, i+1, g_iCPStartFrame[style][i], style);
-			SQL_TQuery(g_hDb, SQL_UpdateReplaysTickCallback, szQuery, _, DBPrio_Low);
+			TicksTransaction.AddQuery(szQuery);
 		}
 	}
 	else{
 		for(int i = 0; i < cp_count; i++){
 			Format(szQuery, sizeof(szQuery), sql_updateReplayCPTicks, g_szMapName, i+1, g_iCPStartFrame[style][i], style);
-			SQL_TQuery(g_hDb, SQL_UpdateReplaysTickCallback, szQuery, _, DBPrio_Low);
+			TicksTransaction.AddQuery(szQuery);
 		}
 	}
+
+	SQL_ExecuteTransaction(g_hDb, TicksTransaction, db_TicksTransactionOnSuccess, db_TicksTransactionOnFailure, DBPrio_Low);
```

Comment:
DBPrio_Low is here wrong. Replace it with this:
`SQL_ExecuteTransaction(g_hDb, TicksTransaction, db_TicksTransactionOnSuccess, db_TicksTransactionOnFailure, _, DBPrio_Low);`

Reply:
or `.priority=DBPrio_Low` works too